src/components/JobCard/__test__/JobCard.test.tsx

```diff
@@ -0,0 +1,101 @@
+import { describe, expect, it, vi } from 'vitest'
+import { render } from '@/tests/customRender'
+import { screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import JobCard from '../JobCard'
+
+// Mock data for the job
+const mockJob = {
+    title: 'Software Engineer',
+    company: 'Tech Corp',
+    location: 'Tokyo, Shibuya',
+    salary: '600,000 - 12,000,000',
+    tags: ['React', 'JavaScript', 'Frontend'],
+    logoUrl: 'https://via.placeholder.com/150',
+    publicationDate: '31/01/2025',
```

Comment:
```suggestion
    publicationDate: '31/01/2025',
```

nit: In Japan we'd usually use YYYY/MM/DD